exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -119,14 +119,26 @@ public void testVerseTwelve() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testMultipleVerses() {
-        String expectedVerseOneToThree = "On the first day of Christmas my true love gave to me, " +
+    public void testFirstThreeVerses() {
+        String expectedVersesOneToThree = "On the first day of Christmas my true love gave to me, " +
                 "a Partridge in a Pear Tree.\n\n" +
                 "On the second day of Christmas my true love gave to me, two Turtle Doves, " +
                 "and a Partridge in a Pear Tree.\n\n" +
                 "On the third day of Christmas my true love gave to me, three French Hens, two Turtle Doves, " +
                 "and a Partridge in a Pear Tree.\n";
-        assertEquals(expectedVerseOneToThree, twelveDays.verses(1, 3));
+        assertEquals(expectedVersesOneToThree, twelveDays.verses(1, 3));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testFourthToSixthVerses() {
+        String expectedVersesFourToSix = "On the fourth day of Chrsitmas my true love gave to me, " + 
```

Comment:
Nearly had it, missed a typo here on this line. Need to flip the s and i in Christmas and then it should be good.

Reply:
Alright, thanks @c-thornton!

Reply:
No problem, should hopefully pass all checks this time.